lsp-mode.el

```diff
@@ -3164,12 +3164,20 @@ ID is the request id. "
                      results
                    (lsp--merge-results (-map #'cl-rest results) method)))))))
 
+(defcustom lsp-default-create-error-handler-fn nil
+  "Default error handler customization.
+Handler should give METHOD as argument and return function of one argument ERROR."
+  :type 'function
+  :group 'lsp-mode)
```

Comment:
can you add `:package-version '(lsp-mode . "8.0.1")`

Reply:
sure